Ambiguous reference warnings during build for XML doc comments

While building the project, warnings similar to this one can be seen in the output:

```
Warning: /home/runner/work/Xyaneon.Games.Dice/Xyaneon.Games.Dice/Xyaneon.Games.Dice/Coin.cs(30,51): warning CS0419: Ambiguous reference in cref attribute: 'Die{TFace}.Roll'. Assuming 'Die<TFace>.Roll()', but could have also matched other overloads including 'Die<TFace>.Roll(int)'. [/home/runner/work/Xyaneon.Games.Dice/Xyaneon.Games.Dice/Xyaneon.Games.Dice/Xyaneon.Games.Dice.csproj]
```

These should be cleaned up.

--- a/Xyaneon.Games.Dice/Coin.cs
+++ b/Xyaneon.Games.Dice/Coin.cs
@@ -27,7 +27,7 @@ namespace Xyaneon.Games.Dice
         public Coin(int seed) : base(Sides, seed) { }
 
         /// <summary>
-        /// An alias for the inherited <see cref="Die{TFace}.Roll"/> method
+        /// An alias for the inherited <see cref="Die{TFace}.Roll()"/> method
         /// specific to coins.
         /// </summary>
         /// <returns>A new <see cref="CoinFlipResult"/> value.</returns>
@@ -43,7 +43,7 @@ namespace Xyaneon.Games.Dice
         /// <exception cref="ArgumentOutOfRangeException">
         /// <paramref name="times"/> is less than one.
         /// </exception>
-        /// <seealso cref="Flip"/>
+        /// <seealso cref="Flip()"/>
         public IEnumerable<CoinFlipResult> Flip(int times) => Roll(times);
     }
 }

--- a/Xyaneon.Games.Dice/Die.cs
+++ b/Xyaneon.Games.Dice/Die.cs
@@ -60,7 +60,7 @@ namespace Xyaneon.Games.Dice
         /// </exception>
         /// <remarks>
         /// If a seed is provided to the constructor, then the result of each
-        /// <see cref="Roll"/> call made in sequence will be predetermined by
+        /// <see cref="Roll()"/> call made in sequence will be predetermined by
         /// the seed value provided.
         /// </remarks>
         /// <seealso cref="Die{TFace}.Die(IEnumerable{TFace})"/>
@@ -97,7 +97,7 @@ namespace Xyaneon.Games.Dice
         /// <returns>The result of the die roll as a <typeparamref name="TFace"/> value.</returns>
         /// <remarks>
         /// If a seed was provided to the constructor, then the result of each
-        /// <see cref="Roll"/> call made in sequence will be predetermined by
+        /// <see cref="Roll()"/> call made in sequence will be predetermined by
         /// the seed value provided.
         /// </remarks>
         /// <seealso cref="Roll(int)"/>
@@ -115,7 +115,7 @@ namespace Xyaneon.Games.Dice
         /// <exception cref="ArgumentOutOfRangeException">
         /// <paramref name="times"/> is less than one.
         /// </exception>
-        /// <seealso cref="Roll"/>
+        /// <seealso cref="Roll()"/>
         public IEnumerable<TFace> Roll(int times)
         {
             if (times < 1)

